dapp/src/components/TransactionModal/ActiveStakingStep/StakeFormModal/StakeDetails.tsx

```diff
@@ -48,14 +38,12 @@ function StakeDetails({ currency }: { currency: CurrencyType }) {
           currency: "usd",
         }}
       />
+
       <TransactionDetailsAmountItem
-        label="Approximate staked tokens"
+        label="You will deposit"
         from={{
           currency,
-          amount: details.estimatedAmount,
-        }}
-        to={{
-          currency: "usd",
+          amount: details.amount,
```

Comment:
Shouldn't we use the amount minus the fee here - `estimatedAmount`?

```suggestion
          amount: details.estimatedAmount,
```

Reply:
Ref commit: 11bc83b7